AudioLibrary.WMME/WaveOutBuffer.cs

```diff
@@ -0,0 +1,108 @@
+using System;
+using System.Runtime.InteropServices;
+using System.Threading;
+using AudioLibrary.WMME.Native;
+
+namespace AudioLibrary.WMME
+{
+    internal class WaveOutBuffer : IDisposable
+    {
+        private readonly object _lockObj = new object();
+
+        public WaveOutBuffer NextBuffer;
+
+        private AutoResetEvent m_PlayEvent = new AutoResetEvent(false);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Make 'm_PlayEvent' 'readonly'. ([csharpsquid:S2933](https://sonarcloud.io/organizations/alex-yevtushenko3820/rules#rule_key=csharpsquid%3AS2933))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=IPP&pullRequest=7&issues=AWfG1-gu8DroMTOoScWo&open=AWfG1-gu8DroMTOoScWo)

[ISSUE_KEY]: AWfG1-gu8DroMTOoScWo